src/styles.scss

```diff
@@ -203,3 +203,54 @@ body {
   flex-direction: column;
   overflow-x: hidden !important;
 }
+
+.table {
+  width: 100%;
+  table-layout: fixed;
+}
+
+.table td {
+  table-layout: fixed;
+  border: 1px solid #ddd;
+  padding: 8px;
+  text-align: center;
+  vertical-align: middle;
+  align-items: center;
+  justify-content: center;
+}
+
+.table th {
+  border: 1px solid #ddd;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  text-align: center;
+  background-color: #ddd;
+  vertical-align: middle;
+}
+
+.table th:nth-child(1),
+.table td:nth-child(1) {
```

Comment:
Table cells don't need to be resized. The size hint on the table headers should be sufficient.